### PR TITLE
UIRuntime: unnecessary required parameters

### PR DESCRIPTION
**Pull request by vitkutny**

### CODE REVIEW

File: src/Bridges/ApplicationLatte/UIRuntime.php [hunk 1]
```diff
@@ -18,11 +18,11 @@
 class UIRuntime extends Nette\Object
 {
 
-	public static function renderSnippets(UI\Control $control, \stdClass $local, array $params)
+	public static function renderSnippets(UI\Control $control, \stdClass $local = NULL, array $params = [])
 	{
 		$control->snippetMode = FALSE;
 		$payload = $control->getPresenter()->getPayload();
-		if (isset($local->blocks)) {
+		if ($local && isset($local->blocks)) {
```
Comment (Majkl578) [inline src/Bridges/ApplicationLatte/UIRuntime.php]: This is actually not needed, isset covers it already: https://3v4l.org/EdJa5

Comment (vitkutny) [inline src/Bridges/ApplicationLatte/UIRuntime.php]: Thank you, updated

**Comment by dg** [comment]
Why?

**Comment by vitkutny** [comment]
I know it's marked as `@internal`. Regardless I'm using it for custom rendering. After all implementation of that method really does not require those params and it won't affect anyone else, I think.
snimek obrazovky 2016-01-15 v 9 50 04

**Pull request merged**